ATTRIBUTIONS-CPP.md

```
@@ -770,6 +770,134 @@ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 THE SOFTWARE.
 ```
 
+## libcurl - 8.5.0
+
+- **Repository URL**: https://github.com/curl/curl
+- **License URL**: https://curl.se/docs/copyright.html
+- **License name**: curl (MIT/X-style)
+- **Usage**: HTTP/HTTPS client library (vendored into wheel, used by AWS SDK and Azure SDK)
+### License Text:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)